components/automate-ui/src/app/components/projects-dropdown/projects-dropdown.component.ts

```diff
@@ -49,8 +53,12 @@ export class ProjectsDropdownComponent implements OnInit, OnChanges {
     }
   }
 
-  ngOnChanges(): void {
+  ngOnChanges(changes): void {
     this.updateLabel();
+    // only update the projects list on initialization/the first change
+    if (changes.projects && changes.projects.firstChange) {
+      this.filteredProjects = this.projectsArray;
+    }
```

Comment:
@msorens - Can you help me understand if I scoped this correctly.  My goal here was to limit the updating of the projects list to only the first change, because unless the page has been reloaded there would be no other new projects coming in.

Reply:
Yes, I believe so. I will take a deeper pass tonight to double check.